proto/ecosystem-incentive/ecosystem_incentive.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package ununifi.ecosystemincentive;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/ecosystem-incentive/types";
+
+message IncentiveUnit {
+  string id = 1 [
+    (gogoproto.moretags) = "yaml:\"id\""
+  ];
+  repeated SubjectInfo subject_info_list = 2 [
+    (gogoproto.moretags) = "yaml:\"subject_info_lists\"",
```

Comment:
more tag and variable name mismatch. subject_info_lists and subject_info_list

Reply:
thanks, i modified it.